src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/InterestRatesController.java

```diff
@@ -36,4 +51,14 @@ public InterestAmount calculateInterest(
             throw new BadRequestException(e.getMessage(), e);
         }
     }
+
+    @GetMapping("/calculate/{externalId:" + UUID_PATTERN + "}")
+    @ApiOperation("Calculates the interest for given external id")
+    public InterestAmount calculateInterest(@PathVariable("externalId") String externalId,
+                                            @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation) {
+
+        Claim claim = claimService.getClaimByExternalId(externalId, authorisation);
+
+        return new InterestAmount(TotalAmountCalculator.calculateInterestForClaim(claim));
```

Comment:
why don't you add an interest field to the claim instead?
it already has a total amount field, would seem more consistent to add it there

Reply:
agree that make more sense. will update

Reply:
done